partiql-ast/src/main/java/org/partiql/ast/v1/expr/ExprValues.java

```diff
@@ -12,14 +12,15 @@
 
 /**
  * TODO docs, equals, hashcode
+ *  Also may not be an [Expr]?
```

Comment:
(self-review): at some point before v1, need to take a closer look at the `VALUES` modeling. I'm not sure yet if it should be an `Expr` or something different.

Reply:
Subqueries generally need to be properly supported. https://github.com/partiql/partiql-lang-kotlin/issues/1641